shared/panels/_flows/reward-flow/_panels/_components/user-contributions/user-contributions.tsx

```diff
@@ -225,8 +225,8 @@ export function UserContributions({ githubUserId, containerHeight = undefined }:
         </nav>
 
         {containerHeight ? (
-          <div className={"overflow-hidden"} style={{ height: containerHeight }}>
-            <ScrollView>{renderContributions()}</ScrollView>
+          <div className={"overflow-hidden"} style={{ maxHeight: containerHeight }}>
+            <ScrollView style={{ maxHeight: containerHeight }}>{renderContributions()}</ScrollView>
```

Comment:
Rename `containerHeight` to `maxHeight` ?

Reply:
i think we should keep maxHeight to be generic